### DISCUSSION TITLE
SSM-U01 Aqara T1 with neutral not reporting power consumption

### DISCUSSION THREAD

**Discussion by vlebourl** [Koenkk/zigbee2mqtt#20312] [Devices]
Hi, I start here before opening an issue as I'm not yet convinced the problem is not on my side. 

I have an Aqara T1s that does not report any power or energy consumption. I have tried several versions of z2m, up to latest 1.34.0-dev commit: [af8c2d8](https://github.com/Koenkk/zigbee2mqtt/commit/af8c2d8) with a sonoff coordinator (zStack3x0 20230507). I have had this device for a while (a year or so) and it never reported power usage... 

How can I find whether the device is actually sending the info? What can I do to get the power measurement feature?
Any help would be very very welcomed!

Here are some more details while the device plugged to the T1 is active and drawing about 800W:

![image](https://github.com/Koenkk/zigbee2mqtt/assets/5701372/585c012b-7dbb-402d-b9ce-1d21af9ce9ef)
![image](https://github.com/Koenkk/zigbee2mqtt/assets/5701372/95f71ae0-788b-4ed9-90a1-cab6b3de5f04)
![image](https://github.com/Koenkk/zigbee2mqtt/assets/5701372/a76ed179-83e5-4622-a686-e596c47c4da1)
![image](https://github.com/Koenkk/zigbee2mqtt/assets/5701372/12b0a128-1df0-4d63-885e-ba3c5aae94b7)

**Comment by vlebourl** [comment]
some debug logging:
```
Zigbee2MQTT:debug 2023-12-20 11:18:17: Received Zigbee message from 'Fil Pilote Lenaic', type 'attributeReport', cluster 'genDeviceTempCfg', data '{"currentTemperature":21}' from endpoint 1 with groupID 0
Zigbee2MQTT:info  2023-12-20 11:18:17: MQTT publish: topic 'zigbee2mqtt_casa/Fil Pilote Lenaic', payload '{"consumption":0,"current":0,"device_temperature":21,"energy":0,"linkquality":91,"power":0,"power_outage_count":0,"power_outage_memory":true,"state":"OFF","switch_type":"toggle","update":{"installed_version":23,"latest_version":23,"state":"idle"},"update_available":null,"voltage":236.49}'
Zigbee2MQTT:debug 2023-12-20 11:18:23: Received Zigbee message from 'Fil Pilote Lenaic', type 'attributeReport', cluster 'aqaraOpple', data '{"247":{"data":[100,16,0,3,40,21,152,57,0,0,0,0,149,57,0,0,0,0,150,57,0,58,19,69,151,57,0,0,0,0,5,33,1,0,154,32,0,11,32,0,8,33,23,1,9,33,0,1],"type":"Buffer"}}' from endpoint 1 with groupID 0
Zigbee2MQTT:info  2023-12-20 11:18:23: MQTT publish: topic 'zigbee2mqtt_casa/Fil Pilote Lenaic', payload '{"consumption":0,"current":0,"device_temperature":21,"energy":0,"linkquality":91,"power":0,"power_outage_count":0,"power_outage_memory":true,"state":"OFF","switch_type":"toggle","update":{"installed_version":23,"latest_version":23,"state":"idle"},"update_available":null,"voltage":235.56}'
```

**Comment by vlebourl** [comment]
I would truly love to get some help or advice. Anyone? 
Cheers :)

**Reply by ildar170975** [reply]
Was it solved?

**Reply by vlebourl** [reply]
Nope

**Reply by Lmhog** [reply]
I have the same problem. For me it worked correctly the first 20 hours, after that, only on/off worked. Tried to delete and join again. Same issues.
